lineage.cue

```diff
@@ -109,17 +109,17 @@ import (
 	//		}
 	//	}
 
-	_schemasAreOrdered: [ for i, sch in SS {
-		if i > 0 {
-			[
-				// sequence is ok if minor version bump
-				if (sch.version[0] == SS[i-1].version[0]) && (sch.version[1] == SS[i-1].version[1]+1) {true},
-				// or if major version bump, minor back at 0
-				if (sch.version[0] == SS[i-1].version[0]+1) && (sch.version[1] == 0) {true},
-				false,
-			][0] & true
-		}
-	}]
+//	_schemasAreOrdered: [ for i, sch in SS {
+//		if i > 0 {
+//			[
+//				// sequence is ok if minor version bump
+//				if (sch.version[0] == SS[i-1].version[0]) && (sch.version[1] == SS[i-1].version[1]+1) {true},
+//				// or if major version bump, minor back at 0
+//				if (sch.version[0] == SS[i-1].version[0]+1) && (sch.version[1] == 0) {true},
+//				false,
+//			][0] & true
+//		}
+//	}]
```

Comment:
Is this still needed? I don't think it is

Reply:
We are only check this here. So I think that we can delete it.

Reply:
At the very least we'll need to figure out another way to have such information at CUE level without breaking, but if we don't find it useful (at least for now), we can get rid of it!

Reply:
@sdboyer any opinion reg this thing? thanks

Reply:
👍🏻